acceptance/lib/facter/acceptance/base_fact_utils.rb

```diff
@@ -0,0 +1,519 @@
+        end
+        results
+      end
+
+      # AIX
+      def aix_expected_facts(agent)
+        version = agent['platform'].match(/aix-(\d)\.(\d)/)
+        kernel_major_version = "#{version[1]}#{version[2]}00"
+        kernel_release = /^#{kernel_major_version}-\d+-\d+-\d+/
+        os_arch     = /[Pp]ower[Pp][Cc]/
+        os_hardware = /IBM/
+
+        expected_facts = {
+            'os.architecture'   => os_arch,
+            'os.family'         => 'AIX',
+            'os.hardware'       => os_hardware,
+            'os.name'           => 'AIX',
+            'os.release.full'   => kernel_release,
+            'os.release.major'  => kernel_major_version,
+            'processors.count'  => /[1-9]+/,
+            'processors.isa'    => os_arch,
+            'processors.models' => os_arch,
+            'kernel'            => 'AIX',
+            'kernelrelease'     => kernel_release,
+            'kernelversion'     => kernel_major_version,
+            'kernelmajversion'  => kernel_major_version
+        }
+        expected_facts
+      end
+
+      # Cisco
+      def cisco_expected_facts(agent)
+        case agent['platform']
+          when /cisco_ios/
+            os_version         = '6'
+            os_name            = 'ios_xr'
+          when /cisco_nexus/
+            os_version         = '7'
+            os_name            = 'nexus'
+          else
+            fail_test "Unknown cisco version #{agent['platform']}"
+        end
+        os_arch                 = 'x86_64'
+        os_hardware             = 'x86_64'
+        processor_model_pattern = 'Westmere'
+
+        expected_facts = {
+            'os.architecture'          => os_arch,
+            'os.family'                => 'cisco-wrlinux',
+            'os.hardware'              => os_hardware,
+            'os.name'                  => os_name,
+            'os.release.full'          => /#{os_version}\.\d(\.\d)?/,
+            'os.release.major'         => os_version,
+            'os.release.minor'         => /\d/,
+            'processors.count'         => /[1-9]/,
+            'processors.physicalcount' => /[1-9]/,
+            'processors.isa'           => /unknown|#{os_hardware}/,
+            'processors.models'        => /#{processor_model_pattern}/,
+            'kernel'                   => 'Linux',
+            'kernelrelease'            => /\d+\.\d+\.\d+/,
+            'kernelversion'            => /\d+\.\d+\.\d+/,
+            'kernelmajversion'         => /\d+\.\d+/
+        }
+        expected_facts
+      end
+
+      # Cumulus
+      def cumulus_expected_facts(agent)
+        os_version       = agent['platform'].match(/cumulus-(\d+\.\d+)-/)[1]
+        os_release_major = os_version.match(/(\d+).\d+/)[1]
+        os_release_minor = os_version.match(/\d+.(\d+)/)[1]
+        if agent['platform'] =~ /cumulus-2\.5-/
+          codename  = 'wheezy'
+        else
+          fail_test "Unknown Cumulus version #{agent['platform']}"
```

Comment:
Cumulus 3.3 is available and based on Debian Jessie according to [their documentation](https://docs.cumulusnetworks.com/display/DOCS/Cumulus+Linux+User+Guide). This may break external testers.